### ISSUE TITLE
▲yPR incorrectly showing as '1X'

### ISSUE THREAD

**Issue by AG-Spike** [thomaschampagne/elevate#804]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: 6.10.2 Stable
  * Chrome/Opera/Chromium version: 73.0.3683.86 (Official Build) (64-bit)
  * OS version: Win 10 (10.0.10586 Build 10586)
  
### Step 3: Describe the problem:
The delta year personal records on a number of segments, but not all in a ride, is showing as '1X' when I have cycled them 10+ this year already. It seems limited to the one ride / activity, and on about a fifth of them? (https://www.strava.com/activities/2247692997)
The same segments are not '1X' in a recent ride and before.

#### Steps to reproduce:

  1. Open my most recent ride (https://www.strava.com/activities/2247692997)
  2. ▲yPRs load up, some missing as 1X
  3. _____
  
#### Actual Behavior:

2jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23 A parser-blocking, cross site (i.e. different eTLD+1) script, https://www.google.com/uds/?file=gdata&v=1.x, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
google.loader.f @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23
2jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23 A parser-blocking, cross site (i.e. different eTLD+1) script, https://www.google.com/uds/api/gdata/1.10/208a907929e7936990d2bf48205aa8d8/core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
google.loader.f @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23
(unknown) Skipping WindyTyModifier execution, no baryCenterPosition available
(unknown) Compute with AthleteModel {"gender":"men","athleteSettings":{"maxHr":188,"restHr":65,"lthr":{"cycling":null,"default":null,"running":null},"cyclingFtp":243,"runningFtp":null,"swimFtp":null,"weight":70}}
(unknown) Using stream cache for activity 'Ifield to Forest Row Ride' (id:2247692997)
chrome-extension://jlgbeobeddnoijopjdaadgomcmmlbjni/strava_celsius.js:43 Changing temps
(unknown) A previous sync exists on Fri Mar 29 2019 09:02:43 GMT+0000 (Greenwich Mean Time)
(unknown) Fast sync mode enabled
(unknown) Local and remote activities count matches.
(unknown) Last sync date time saved:  Fri Mar 29 2019 09:04:53 GMT+0000 (Greenwich Mean Time)
(unknown) Fast sync finished Object
  
#### Expected Behavior:
To have a comparison / change to my previous year's best time.
  
#### Media attachement:
What you see in the screen shots, all the 1X should have values.
![Untitled2](https://user-images.githubusercontent.com/48119010/55221803-47d40400-5202-11e9-9911-67e399e64c17.jpg)
![Untitled1](https://user-images.githubusercontent.com/48119010/55221804-47d40400-5202-11e9-921a-200ac62a8306.jpg)

#### Providing your Elevate synced athlete activities can be useful to solve the problem
[2019.3.29-9.10_v6.10.2.history.zip](https://github.com/thomaschampagne/elevate/files/3021866/2019.3.29-9.10_v6.10.2.history.zip)
